lib/Charge.php

```diff
@@ -79,6 +79,8 @@ public function capture($params = null, $options = null)
      * @param array|null $params
      * @param array|string|null $options
      *
+     * @deprecated Use the `save` method on the Dispute object
+     *
```

Comment:
Does `$this->dispute` load an actual `Dispute` now? Do we want to just glue things through as one would expect: update the `Dispute` with all of the `$params` and call `save` on it?